jekyll/_cci2/monitoring.adoc

```diff
@@ -576,6 +576,95 @@ Custom Metrics using a Telegraf configuration file may be configured in addition
 .Custom Metrics
 image::metrics-custom.png[Custom]
 
+=== Available Metrics
```

Comment:
@GERey custom metrics info here for review and completion when you have all the metrics descriptions